Mathlib/NumberTheory/NumberField/Units/Basic.lean

```diff
@@ -96,6 +96,10 @@ protected theorem norm [NumberField K] (x : (𝓞 K)ˣ) :
     |Algebra.norm ℚ (x : K)| = 1 := by
   rw [← RingOfIntegers.coe_norm, isUnit_iff_norm.mp x.isUnit]
 
+@[simp]
```

Comment:
Do we really want to tag `simp` an inequality like this?

Reply:
You're right. It's probably not a good idea.